src/modules/pipeline/worker.c

```diff
@@ -246,7 +246,7 @@ void ecs_workers_progress(
     }
 
     if (world->measure_frame_time) {
-        world->stats.system_time_total += (float)ecs_time_measure(&start);
+        world->stats.system_time_total += (double)ecs_time_measure(&start);
```

Comment:
The return type of `ecs_time_measure` is already a double, cast is not necessary.